.gitlab-ci-check-golang-acceptance.yml

```diff
@@ -0,0 +1,110 @@
+# .gitlab-ci-check-golang-acceptance.yml
+#
+# This gitlab-ci template build, runs and publises into codecov acceptance
+# tests from a Go repository.
+#
+# It assumes a tests/ folder with a Dockerfile for the Python test runner,
+# and a Dockerfile.acceptance-testing for the Go test binary.
+#
+# Add it to the project in hand through Gitlab's include functionality
+#
+# include:
+#   - project: 'Northern.tech/Mender/mendertesting'
+#     file: '.gitlab-ci-check-golang-acceptance.yml'
+#
+# The template uses latest golang version. To override the version
```

Comment:
Nitpick: `1.13` is latest not, but not for as long as this comment will live probs

Reply:
I don't understand your comment.

Reply:
sry. The jist of it is. What happens when they release go `1.14` ?

Reply:
I see, thanks.

I will rephrase a bit the comment (as using the actual latest with `golang:latest` is very dangerous`).